src/su.c

```diff
@@ -676,12 +676,12 @@ static /*@only@*/struct passwd * do_check_perms (void)
 	if (strcmp (name, tmp_name) != 0) {
 		SYSLOG ((LOG_INFO,
 		         "Change user from '%s' to '%s' as requested by PAM",
-		         name, tmp_name));
+		         name, (const char *) tmp_name));
```

Comment:
Instead of casting, can we declare the variable of type `const char *`?

Reply:
<https://github.com/shadow-maint/shadow/pull/874>

Reply:
Since we merged the change of type, please drop this patch.

Reply:
Dropped commit.